streams/src/test/java/org/apache/kafka/streams/state/internals/SessionStoreBuilderTest.java

```diff
@@ -118,48 +114,28 @@ public void shouldHaveCachingAndChangeLoggingWhenBothEnabled() {
     }
 
     @Test
-    public void shouldThrowNullPointerIfInnerIsNull() {
+    public void shouldThrowNullPointerIfSupplierIsNull() {
         final Exception e = assertThrows(NullPointerException.class, () -> new SessionStoreBuilder<>(null, Serdes.String(), Serdes.String(), new MockTime()));
         assertThat(e.getMessage(), equalTo("storeSupplier cannot be null"));
     }
 
     @Test
-    public void shouldThrowNullPointerIfKeySerdeIsNull() {
-        final Exception e = assertThrows(NullPointerException.class, () -> new SessionStoreBuilder<>(supplier, null, Serdes.String(), new MockTime()));
-        assertThat(e.getMessage(), equalTo("name cannot be null"));
-    }
-
-    @Test
-    public void shouldThrowNullPointerIfValueSerdeIsNull() {
-        final Exception e = assertThrows(NullPointerException.class, () -> new SessionStoreBuilder<>(supplier, Serdes.String(), null, new MockTime()));
```

Comment:
Ah, this looks like the same issue as https://github.com/apache/kafka/pull/14152#discussion_r1284245654